scylladb/sphinx-scylladb-theme
# Add community forum icon

Closes #645 

## How to test this PR

1. Clone this PR.
2. Go to the docs folder.
2. Run ``make preview``
3. Open http://localhost:5500/
4. You should see the new community forum icon:

![image](https://user-images.githubusercontent.com/9107969/206440610-5c98fbe7-af83-4a87-b1a1-3c2dd908a05b.png)

Note that I've also changed the user mailing list icon so that they share the same style.
